lib/mumukit/login/provider/auth0.rb

```diff
@@ -41,7 +41,7 @@ def auth0_config
   end
 
   def lock_settings(controller, login_settings, options)
-    login_settings.to_lock_json(controller.url_for(callback_path), options)
+    locale = controller.current_organization.locale rescue 'en'
```

Comment:
Please avoid local variables when a method is suitable. Also, there is no need to add `current_platform` to `controller` since the `Mumukit::Platform::Organization.current` is global. 

Lastly, please prefer nil-safe access to rescue expressions. For example: 

```
def current_locale
    Mumukit::Platform::Organization.current&.locale || 'en'
end
```